### PR TITLE
Fix order

### LINKED ISSUE #4025: Assignement of labels to frames occurs in reveresed order
### My actions before raising this issue
- [✓] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [✓] Searched [past issues](/issues)

### Expected Behaviour

Using the web interface at https://cvat.org/, when exporting task dataset, the annotations should be assigned to the frames as they are shown in the interface.

### Current Behaviour

When loading the images into the interface by selecting them all from a given directory, the uploading occurs without problems and they are shown in the correct order in the interface. However, when exporting the task dataset, labels are assigned to frames in reversed order, making the annotations unusable.  This does not seem to occur when a video is loaded instead of a batch of separate images.

### Steps to Reproduce (for bugs)

This link shows a sequence loaded in the right order but when you export with the "save the images" option clicked, the images are saved in reversed order. In other words, frame one contains last frame:. 
https://cvat.org/tasks/154079/jobs/234353

### Your Environment

- Operating System and version (e.g. Linux, Windows, MacOS): Ubuntu 18.04

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

### PR DESCRIPTION
**Pull request by Marishka17**
### Motivation and context

Resolve #4025
### How has this been tested?

Manually
### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by zhiltsov-max** [comment]
Could you check [these lines](https://github.com/openvinotoolkit/cvat/blob/ff5b48797fc3418ca623db335526888cc9fc77b9/cvat/apps/dataset_manager/bindings.py#L379-L382)? If the results are in the reversed order, it looks like the `annotation_ir` is reversed. I'm not sure that the order in `frame_info` is a valid (or reliable) fix.

File: cvat/apps/dataset_manager/task.py [hunk 1]
```diff
@@ -547,7 +548,9 @@ def import_annotations(self, src_file, importer):
 
 class TaskAnnotation:
     def __init__(self, pk):
-        self.db_task = models.Task.objects.prefetch_related("data__images").get(id=pk)
+        self.db_task = models.Task.objects.prefetch_related(
+            Prefetch('data__images', queryset=models.Image.objects.order_by('frame'))
+        ).get(id=pk)
 
         # Postgres doesn't guarantee an order by default without explicit order_by
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/task.py:555]: It seems this comment also applies to the current fix.

**Review by zhiltsov-max — APPROVED** [review]
I'm not sure I'm the right person to review questions related to CVAT DB use, but for me the current fix looks more or less valid. If you see how we can enforce annotation ordering independently of DB requests, I'd prefer such solution.

**Review by nmanovic — DISMISSED** [review]
LGTM

**Pull request merged**